style/pi-hole.css

```diff
@@ -330,3 +330,13 @@
 .small-box {
   border-radius: 10px;
 }
+
+/* Work around code to make the scaling of SVG icons in the
+   top row on the dashboard much more natural. I'll post a
+   pull request to the parent theme, so this should only be
+   needed temporarily */
```

Comment:
Is this PR still valid concerning your comment that this would only be needed temporarily? If so, please address the complaint by @DCO, see here

https://github.com/pi-hole/AdminLTE/pull/1748/checks?check_run_id=1946538312

Reply:
We would still need to update adminLTE even without this. I doubt they are doing any more adminLTE 2.x releases...